dicoogle/src/main/java/pt/ua/dicoogle/core/settings/part/ArchiveImpl.java

```diff
@@ -76,6 +77,9 @@ public static ArchiveImpl createDefault() {
     @JsonProperty("watch-directory")
     private String watchDirectory;
 
+    @JsonProperty("support-wsi")
```

Comment:
What happens if this element is missing in server.xml? If it blows up, then we need to give it a default.
```suggestion
    @JsonProperty("support-wsi", defaultValue = "false")
```